docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
+
+In constrast, ratio-based constraints consider the ratio as opposed to the
+difference. Parity is represented as
+
+.. math::
+
+    r \leq \dfrac{\text{metric}_a}{\text{metric}_*} \leq \dfrac{1}{r} \; \forall a
+
+with a :code:`ratio_bound` :math:`r`. Similar to the :code:`difference_bound`
+for difference-based constraints, ratio-based constraints have a
+:code:`ratio_bound_slack`. The two inequalities can also be considered
+separately as
+
+.. math::
+
+   r \leq \dfrac{\text{metric}_a}{\text{metric}_*} \; \forall a \\
+   \dfrac{\text{metric}_a}{\text{metric}_*} \leq \dfrac{1}{r} \; \forall a
+
+and one step further in the equivalent form:
+
+.. math::
+
+   - \text{metric}_a + r \cdot \text{metric}_* \leq 0 \; \forall a \\
+   r \cdot \text{metric}_a - \text{metric}_* \leq 0 \; \forall a
+
+To use a constraint in its ratio-based version simply indicate as much through
+its arguments
+
+    >>> ClassificationMoment(ratio_bound=0.9, ratio_bound_slack=0.01)
+
+In other words, this means that
+:math:`0.9 \cdot \text{metric}_a \leq \text{metric}_*`, i.e., the metric value for
+each group needs to be at least 90% of the overall metric value, and
+:math:`0.9 \cdot \text{metric}_* \leq \text{metric}_a`, i.e., the overall metric
+value needs to be at least 90% of each group's metric value.
+
+.. note::
+
+    It is not possible to specify both :code:`difference_bound` *and*
+    :code:`ratio_bound` for the same constraint object.
+
+.. _demographic_parity:
+
+Demographic Parity
+^^^^^^^^^^^^^^^^^^
+
+A classifier :math:`h(X)` satisfies *Demographic Parity* if
+
+.. math::
+    
+    \P[h(X) = y' \given A = a] = \P[h(X) = y'] \; \forall a, y'
+ 
+In other words, the selection rate or percentage of samples with label 1
+should be equal between all groups. Implicitly this means the percentage
+with label 0 is equal as well.
+
+In the example below class :code:`"a"` has a selection rate of 60%,
+:code:`"b"` has a selection rate of 20%. The overall selection rate is 40%,
+so :code:`"a"` is `0.2` above the overall selection rate, and :code:`"b"` is
+`0.2` below. The seemingly redundantly stored constraint violation with both
+:code:`"a"` and :code:`"b"` showing up twice stems from the constraint
+definition outlined in :ref:`constraints_binary_classification`. Each equality
+constraint is written as two inequalities of which one contains
+:math:`\text{metric}_a` positively (sign :math:`+`) and one negatively
+(sign :math:`-`).
+Note that the specified :code:`difference_bound` is not used
+since the constraint object is not used in a mitigation in this example.
+Similarly, :code:`y_true` is technically irrelevant to the calculations
+because the underlying metric of Demographic Parity, selection rate, does not
+consider performance relative to the true labels, but rather proportions in
+the predicted labels.
+
+.. note::
+
+    In mitigation examples we never need to call :code:`load_data` or
```

Comment:
You haven't mentioned `load_data()` to this point. This is also skipping into implementation when I thought this section was about the theory behind the moments.

Reply:
Correct. It's because I don't want to actually talk about load_data or gamma, because they are irrelevant to users. This is a user guide and therefore won't have much theory behind moments other than a definition, which is provided above the note. I feel like you're expecting something more?

Reply:
It doesn't make sense to say that you're not calling something when you haven't mentioned that it exists yet.

Reply:
I slightly rearranged, but there is space for improvements (not sure whether they need to happen in this PR).

Reply:
Agreed, there will be plenty more we can do over time, but *something* is better than nothing to start with.